k8s/kube-prometheus/conbench-grafana-dashboard.json

```diff
@@ -1207,6 +1207,39 @@
         "skipUrlSync": false,
         "type": "datasource"
       },
+      {
+        "current": {
+          "selected": true,
+          "text": "vd-2",
+          "value": "vd-2"
+        },
+        "hide": 0,
+        "includeAll": false,
+        "label": "cluster",
+        "multi": false,
+        "name": "cluster",
+        "options": [
+          {
+            "selected": true,
+            "text": "vd-2",
+            "value": "vd-2"
+          },
+          {
+            "selected": false,
+            "text": "ursa-2",
+            "value": "ursa-2"
+          },
+          {
+            "selected": false,
+            "text": ".*",
+            "value": ".*"
+          }
+        ],
+        "query": "vd-2, ursa-2, .*",
```

Comment:
Ideally this can be made more dynamic. Looking up label values from the DB works (I used that method in a first iteration), but for such a variable one cannot inject a custom value (a value that is not part of a query result). There is a special value "all", but that doesn't include "not set". So, see above, I resorted to using a regex match and the special value .* includes "not set".